BraveWallet/Crypto/Asset Details/AssetDetailHeaderView.swift

```diff
@@ -45,6 +46,62 @@ struct AssetDetailHeaderView: View {
     // About 300 points added so it doesn't animate funny
     (0..<300).map { _ in .init(date: Date(), price: "0.0") }
   }
+  
+  private var isBuySupported: Bool {
+    assetDetailStore.isBuySupported
+    && WalletConstants.supportedBuyWithWyreNetworkChainIds.contains(networkStore.selectedChainId)
+  }
+  
+  @ViewBuilder var buySendSwapButtonsContainer: some View {
```

Comment:
nit: I think it might be cleaner if we use two computed properties for this similar to `SuggestedNetworkView` / `AddSuggestedTokenView`:
- `@ViewBuilder actionButtons: some View`
- `@ViewBuilder actionButtonsContainer: some View`

Then we only need to write out the code for each button once, and the main `body: some View` doesn't need to hold any of the aurora logic.